proto/streamlit/proto/Selectbox.proto

```diff
@@ -27,6 +27,7 @@ message Selectbox {
   string form_id = 6;
   int32 value = 7;
   bool set_value = 8;
-  bool disabled = 9;
-  LabelVisibilityMessage label_visibility = 10;
+  string placeholder = 9;
+  bool disabled = 10;
+  LabelVisibilityMessage label_visibility = 11;
```

Comment:
We'll have to add the new `string placeholder` field as tag number 11 here. Due to some external dependencies, we now have to maintain backwards compatibility between our protobuf schema versions, which means that renumbering the `disabled` and `label_visibility` fields is an illegal change.

Reply:
https://github.com/streamlit/streamlit/pull/6913/commits/f2108be5a3de8c64bf808323b8e5d993c391d8c2
Thanks for letting me know about the external dependencies. I have fixed it in the above commit.